File: ruby/lib/grandiorite.rb
require "rainbow"

class Grandiorite
  def self.hello_world
    puts [
      Rainbow("H").red,
      Rainbow("e").red.bright,
      Rainbow("l").orange,
      Rainbow("l").orange.bright,
      Rainbow("o").yellow,
      Rainbow(",").yellow.bright,
      Rainbow(" ").green,
      Rainbow("W").green.bright,
      Rainbow("o").blue,
      Rainbow("r").blue.bright,
      Rainbow("l").purple,
      Rainbow("d").purple.bright,
      Rainbow(".").white,
    ].join
  end
end

File: erlang/lib/grandiorite.ex
defmodule Grandiorite do
  @moduledoc """
  Documentation for `Grandiorite`.
  """

  @doc """
  Hello world in ANSI colors.

  ## Examples

      iex> Grandiorite.helloWorld()
      "Hello, World."

  """
  def helloWorld do
    [:red, "H", :reset,
    :red, :bright, "e", :reset,
    :yellow, :faint, "l", :reset,
    :yellow, :bright, "l", :reset,
    :green, :faint, "o", :reset,
    :green, :bright, ",", :reset,
    :cyan, :faint, " ", :reset,
    :cyan, :bright, "W", :reset,
    :blue, :faint, "o", :reset,
    :blue, :bright, "r", :reset,
    :magenta, :faint, "l", :reset,
    :magenta, :bright, "d", :reset,
    :white, :faint, ".", :reset,
    ]
      |> IO.ANSI.format
      |> IO.puts
  end
end
